components/monitoring/README.md

```diff
@@ -41,159 +41,117 @@ for adding the service monitor declaration:
-      name: grafana-dashboard-dora-metrics-volume
-    ```
+ - Check out [this example o11y PR](https://github.com/redhat-appstudio/o11y/pull/39) for creating a dashboard in the team's repository
+  
 
-  - Add a link to the new maps folder in `appstudio-workload-monitoring` [ConfigMap](https://github.com/redhat-appstudio/infra-deployments/blob/2a6e4dcb272fa01d330b393d87b8f5ea5c434687/components/monitoring/grafana/base/grafana-app.yaml#L234) in  `grafana-app.yaml`
-    ```yaml
-    apiVersion: v1
-    kind: ConfigMap
-    metadata:
-      namespace: appstudio-workload-monitoring
-      name: grafana-dashboards
-    data:
-      default.yaml: |
-        apiVersion: 1
-        providers:
-          - name: Dora Metrics
-            folder: QE
-            type: file
-            disableDeletion: true
-            options:
-              path: /var/lib/grafana/dashboards-dora-metrics
-    ```
+#### infra-deployments repository
+Follow the next steps to add the dashborad you just created to infra-deployments
+
+  - Create a team folder under `components/monitoring/grafana/base/<team_name>`
   
-  - Note: Grafana dashboards has to have a predefined datasource name. It is recommended to use templating to select them. For example:
-    ```json
-      "templating": {
-        "list": [
-          {
-            "current": {
-              "selected": true,
-              "text": "prometheus-appstudio-ds",
-              "value": "prometheus-appstudio-ds"
-            },
-            "hide": 0,
-            "includeAll": false,
-            "multi": false,
-            "name": "datasource",
-            "options": [],
-            "query": "prometheus",
-            "queryValue": "",
-            "refresh": 1,
-            "regex": ".*-(appstudio)-.*",
-            "skipUrlSync": false,
-            "type": "datasource"
-          }
-        ]
-      },
+  - Create a kustomization.yaml file and add the dashboard you created as a resource by using the commit sha as ref, for example:
+    ```
+    apiVersion: kustomize.config.k8s.io/v1beta1
+    kind: Kustomization
+    resources:
+        - https://github.com/redhat-appstudio/o11y/grafana/?ref=82bec1c488250ae32d458c77755e432329be1b45
     ```
-    In this example dashboard will use `prometheus-appstudio-ds` datasource, with ability to use other datasource that contains `appstudio` in the name.
     
+  - Add your dashboard to the base [kustomization file](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/monitoring/grafana/base/kustomization.yaml#L14) to automatically add it to future deployments.
+   
+  - Push the code to infra-deployments repository
+
+
+#### Verification
+
+Follow the next steps to To verify and check your dashboard after merging the configurations
+
+- Open the Grafana UI
+
+- Click the `Manage` option in the `Dashboards` menu
 
+- Verify that your team’s dashboard is located under appstudio-grafana folder 
```

Comment:
```suggestion
- Verify that your team’s dashboard is located under `appstudio-grafana` folder 
```
since this is a step-by-step, perhaps we should use numbers instead of dashes

Reply:
Done